specs/phase0/validator.md

```diff
@@ -214,9 +219,18 @@ A validator has two primary responsibilities to the beacon chain: [proposing blo
 
 ### Block proposal
 
-A validator is expected to propose a [`SignedBeaconBlock`](./beacon-chain.md#signedbeaconblock) at the beginning of any slot during which `is_proposer(state, validator_index)` returns `True`. To propose, the validator selects the `BeaconBlock`, `parent`, that in their view of the fork choice is the head of the chain during `slot - 1`. The validator creates, signs, and broadcasts a `block` that is a child of `parent` that satisfies a valid [beacon chain state transition](./beacon-chain.md#beacon-chain-state-transition-function).
+A validator is expected to propose a [`SignedBeaconBlock`](./beacon-chain.md#signedbeaconblock) at
+the beginning of any slot during which `is_proposer(state, validator_index)` returns `True`.
+To propose, the validator selects the `BeaconBlock`, `parent`,
+that in their view of the fork choice is the head of the chain during `slot - 1`.
+The validator creates, signs, and broadcasts a `block` that is a child of `parent`
+that satisfies a valid [beacon chain state transition](./beacon-chain.md#beacon-chain-state-transition-function).
 
-There is one proposer per slot, so if there are N active validators any individual validator will on average be assigned to propose once per N slots (e.g. at 312,500 validators = 10 million ETH, that's once per ~6 weeks).
+There is one proposer per slot, so if there are N active validators any individual validator
+will on average be assigned to propose once per N slots (e.g. at 312,500 validators = 10 million ETH, that's once per ~6 weeks).
+
+*Note*: In this sectio, `state` is the state of the slot for the block proposal _without_ the block yet applied.
```

Comment:
Note! These two lines are the only change.

I started breaking up long lines in the validator guide (like the p2p spec). Will help with diffs in the future but some pain right now